src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String REPORT_TEMPLATE = "Report for period %s - %s%s";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String EMPLOYEE_TEMPLATE = "%s - %d";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
```

Comment:
This method is very complicated and hard to read, please break it down to smaller methods